app/controllers/index.js

```diff
@@ -8,7 +8,8 @@ var db = new PouchDB('./db/targetDB'),
 refDb = new PouchDB('./db/referenceDB'),
```

Comment:
You haven't pulled the changes from master in to your branch. Your branch is not updated.